src/index.tsx

```diff
@@ -39,6 +41,11 @@ type PiwikProSdkType = {
       value?: number;
     }
   ): Promise<void>;
+  setCustomDimension(
+    index: number,
+    value: string,
+    scope: CustomDimensionScope
```

Comment:
I think the `scope` name is a bit dangerous as it is very easily mixed with the server side scope https://help.piwik.pro/support/analytics-new/custom-dimension/#about-the-scope-for-a-custom-dimension which has a similar but significantly different meaning.